src/styles/_base.scss

```diff
@@ -82,4 +82,8 @@ html {
     box-sizing: border-box;
     scroll-behavior: smooth;
     -webkit-overflow-scrolling: touch;
+
+    a {
+        text-decoration: none;
```

Comment:
also having accent colour for all links will make UI more beautiful. Can consult Aadarsha for that.